nextflow/workspace_description.md

```diff
@@ -12,26 +12,26 @@ This notebook shows how to get started with [Nextflow](https://www.nextflow.io)
   1. [`terra-axon-examples` example repo](https://github.com/DataBiosphere/terra-axon-examples): Contains JupyterLab notebook `nextflow_examples.ipynb` which demonstrates how to configure and run Nextflow workflows in Enterprise Terra.
```

Comment:
This line mentions "Enterprise Terra", as well as on line 3 of this file.

Reply:
thanks for catching -- fixed!